docs/sources/clients/lambda-promtail/_index.md

```diff
@@ -48,7 +48,7 @@ provider "aws" {
 }
 ```
 
-To keep the log group label add `-var "keep_stream=true"`.
+To keep the log stream label add `-var "keep_stream=true"`.
```

Comment:
can we make this a bit more verbose, and include ```(`__aws_cloudwatch_log_stream\`)``` so that users have an easy to know which label from cloudwatch this would preserve